src/content/builds/builds.json

```diff
@@ -4,21 +4,21 @@
     {
       "title": "Empathic Accuracy",
       "tags": ["React", "Node.js", "Video", "Emotion", "Psychology"],
-      "image": "http://www.youandthemat.com/wp-content/uploads/nature-2-26-17.jpg",
+      "image": "./empathic.jpg",
```

Comment:
Not sure what happened but all the images are Arun now:
<img width="1262" alt="Screen Shot 2019-05-07 at 8 53 02 AM" src="https://user-images.githubusercontent.com/13836027/57300819-900a0000-70a5-11e9-964f-1ff6f743c5d0.png">